lib/param_parsing/versiontf.go

```diff
@@ -0,0 +1,34 @@
+package param_parsing
+
+import (
+	"github.com/hashicorp/terraform-config-inspect/tfconfig"
+	"github.com/warrensbox/terraform-switcher/lib"
+	"os"
+)
+
+const versionTfFileName = "version.tf"
+
+func GetVersionFromVersionsTF(params Params) Params {
+	filePath := params.ChDirPath + "/" + versionTfFileName
+	if lib.CheckFileExist(filePath) {
+		logger.Infof("Reading version from %q", filePath)
+		module, err := tfconfig.LoadModule(params.ChDirPath)
+		if err != nil {
+			logger.Fatal("Could not load terraform module")
```

Comment:
It may make sense to provide info on which path the module failed to load from. Related to https://github.com/warrensbox/terraform-switcher/pull/361

Reply:
Done